gatekeeper-service/deploy/service.yaml

```diff
@@ -43,4 +45,59 @@ spec:
   - port: 8080
     protocol: TCP
   selector:
-    run: gatekeeper-service
\ No newline at end of file
+    run: gatekeeper-service
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    deployment.kubernetes.io/revision: "1"
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"apps/v1","kind":"Deployment","metadata":{"annotations":{},"name":"gatekeeper-service-evaluation-done-distributor","namespace":"keptn"},"spec":{"replicas":1,"selector":{"matchLabels":{"run":"distributor"}},"template":{"metadata":{"labels":{"run":"distributor"}},"spec":{"containers":[{"env":[{"name":"PUBSUB_URL","value":"nats://keptn-nats-cluster"},{"name":"PUBSUB_TOPIC","value":"sh.keptn.events.evaluation-done"},{"name":"PUBSUB_RECIPIENT","value":"gatekeeper-service"}],"image":"keptn/distributor:latest","name":"distributor","ports":[{"containerPort":8080}],"resources":{"limits":{"cpu":"500m","memory":"128Mi"},"requests":{"cpu":"50m","memory":"32Mi"}}}]}}}}
+  creationTimestamp: "2020-05-08T07:50:00Z"
+  generation: 1
+  name: gatekeeper-service-evaluation-done-distributor
+  namespace: keptn
+  resourceVersion: "594181"
+  selfLink: /apis/extensions/v1beta1/namespaces/keptn/deployments/gatekeeper-service-evaluation-done-distributor
+  uid: 84dff9d8-9100-11ea-9b9a-42010a80002c
+spec:
```

Comment:
did you add the deployment of the distributor on purpose? If so, please remove the metadata like uid, selfLink, resourceVersion, ...

Reply:
that change was unintentional and has been removed